docs/FLYSKY_FRM303.md

```diff
@@ -1,8 +1,8 @@
-# mLRS Documentation: FlySky FRM303 Transmitter Module #
+# mLRS Documentation: FlySky FRM303 RF Module #
```

Comment:
I am not sure I should like or not like this, Throughout the docs we settled for calling the thing which goes into the back of a radio a "transmitter module" or shortened "Tx module". You could make the argument also for the R9M. In fact, for every new occasion one caould repeat the argumentation.

That's just my opinion. Pl sort out with @jlpoltrack. I would suggest however to decide and use with some consistency.

:)

Reply:
I did not revert this and wait now feedback from @jlpoltrack as IMHO `Transmitter Module` is incorrect term here.

Reply:
well, might be technically incorrect, but it appears to be the commonly used and accepted terminus...
https://www.radiomasterrc.eu/product-category/transmitter-modules/

:)

Reply:
My two cents would be to just call it 'FlySky FRM303 Module' like the main readme page as the 'RF' sounds a bit unnecessary to me.

<img width="305" alt="image" src="https://github.com/olliw42/mLRS-docu/assets/41841496/c114149c-5757-4bea-9e88-c14c041cdbf0">